CHANGELOG.md

```diff
@@ -29,6 +37,14 @@ Versioning](https://semver.org/spec/v2.0.0.html).
   - The `sourceId` field in the `export` GraphQL API is renamed to `sensorId`.
   - The `source` field in the filter parameters of all GraphQL APIs is changed
     to `sensor`.
+- Update the compatibility version of the quic communication modules.
+  - Changed `PEER_VERSION_REQ` to ">=0.23.0-alpha.1,<0.24.0".
+  - Changed `INGEST_VERSION_REQ` to ">=0.23.0-alpha.1,<0.24.0".
+  - Changed `PUBLISH_VERSION_REQ` to ">=0.23.0-alpha.1,<0.24.0".
+- Fixed code related to migration.
+  - Changed `COMPATIBLE_VERSION_REQ` to ">=0.23.0-alpha.1,<0.24.0
```

Comment:
Could you check this please?

Reply:
(KR) 혹시나 하여 말씀드리면, migration.rs에서 `COMPATIBLE_VERSION_REQ` 관련 동작 로직을 확인해주실 수 있을까요? 이번 PR에서 `COMPATIBLE_VERSION_REQ`는 >=0.23.0-alpha.2,<0.24.0" 가 되었어야 할 것 같습니다. Version 관련 코멘트가 여러번 나오는 이유가 혹시 단순 실수가 아니라 무언가 놓치고 있는 점 있는 것은 아닌지를 더블체크해주시면 좋겠습니다. migration 오류는 치명적인 타격을 줄 수 있으므로 여러번 확인하는 것이 더 낫다고 생각합니다. 

(EN) Could you maybe review the logic related to `COMPATIBLE_VERSION_REQ` in migration.rs? In this PR, I think `COMPATIBLE_VERSION_REQ` should have been set to ">=0.23.0-alpha.2,<0.24.0". The reason version-related comments keep coming up might not just be due to a simple mistake, but it could be a sign that something is being overlooked regarding version management and migration. It would be helpful if you could double-check whether there’s any detail being missed beyond just a minor oversight. Migration errors can cause severe impact, so I believe it is worth to double or even triple-check thoroughly.

Reply:
테스트 코드 및 최근 데이터 기준으로는 이슈 없었습니다.
예전 giganto version 기준으로 데이터 생성 후 마이그레이션 테스트하고 조회 되는지 체크 해보도록 하겠습니다.
VERSION_REQ도 모두 `>=0.23.0-alpha.2,<0.24.0`로 변경하도록 하겠습니다.